docs/proposals/20191030-machine-health-checking.md

```diff
@@ -204,13 +216,5 @@ This propose the new CRD to belong to the same API group than other cluster-api
 
 ### Version Skew Strategy [optional]
 
-
-## Implementation History
-- [ ] MM/DD/YYYY: Proposed idea in an issue or [community meeting]
-- [ ] MM/DD/YYYY: Compile a Google Doc following the CAEP template (link here)
-- [ ] MM/DD/YYYY: First round of feedback from community
-- [ ] MM/DD/YYYY: Present proposal at a [community meeting]
-- [ ] MM/DD/YYYY: Open proposal PR
-
```

Comment:
I think keeping the Implementation history in some manner here would be helpful to track the change between MHC-initiated deletion and the newer annotation approach.

Reply:
I agree, I was removing this since it was never filled out in the first place. I can retroactively create it, but I was just cleaning this up.